pkg/runctl/secret_manager.go

```diff
@@ -0,0 +1,102 @@
+package runctl
+
+import (
+	"github.com/SAP/stewardci-core/pkg/apis/steward/v1alpha1"
+	"github.com/SAP/stewardci-core/pkg/k8s"
+	secrets "github.com/SAP/stewardci-core/pkg/k8s/secrets"
+	runifc "github.com/SAP/stewardci-core/pkg/runctl/run"
+	"github.com/pkg/errors"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	klog "k8s.io/klog/v2"
+)
+
+type secretManager struct {
+	secretHelper secrets.SecretHelper
+}
+
+// NewSecretManager creates secrets in the run namesapce
+func NewSecretManager(secretHelper secrets.SecretHelper) runifc.SecretManager {
+	return &secretManager{
+		secretHelper: secretHelper,
+	}
+}
```

Comment:
See <https://github.com/golang/go/wiki/CodeReviewComments#interfaces>.

`NewSecretManager()` should return the concrete type `SecretManger` (make it public).

Reply:
See https://github.com/SAP/stewardci-core/pull/165/commits/87569ae719bc0d94f271f5212c92063b503d03a3